### PR TITLE
Removed overly-enthusiastic-about-scala-sample exclamation point in doc

### PR DESCRIPTION
**Pull request by helena**

### CODE REVIEW

**Review by raboof — APPROVED** [review]
:)

File: akka-docs/src/main/paradox/cluster-sharding.md [hunk 1]
```diff
@@ -130,7 +130,7 @@ Java
 @@@ div { .group-scala }
 
 A more comprehensive sample is available in the
-tutorial named [Akka Cluster Sharding with Scala!](https://github.com/typesafehub/activator-akka-cluster-sharding-scala).
+tutorial named [Akka Cluster Sharding with Scala](https://github.com/typesafehub/activator-akka-cluster-sharding-scala).
```
Comment (patriknw) [inline akka-docs/src/main/paradox/cluster-sharding.md]: That is a very! old sample. We should link to a newer sample. https://github.com/akka/akka-samples/tree/2.5/akka-sample-sharding-scala

Look at how we link to such samples in other places. I think the link is https://developer.lightbend.com/start/?group=akka&project=akka-samples-cluster-sharding-scala

There is also a corresponding Java sample

Comment (helena) [reply akka-docs/src/main/paradox/cluster-sharding.md]: Perfect, will update!

**Review by patriknw — APPROVED** [review]
LGTM

**Pull request merged**